extensions/google-cloud-platform-search/.gitignore

```diff
@@ -11,3 +11,10 @@ compiled_raycast_swift
 
 # misc
 .DS_Store
+.idea
+.npmrc
+raycast-env.d.ts
+
+.raycast-swift-build
+.swiftpm
+compiled_raycast_swift
```

Comment:
These are duplicated.

Reply:
Thank you, I missed them during conflict resolution! Removed them ✅